addons/dockable_container/dockable_panel_reference_control.gd

```diff
@@ -26,7 +26,9 @@ func set_reference_to(control: Control) -> void:
 	if _reference_to != control:
 		if _reference_to:
 			_reference_to.disconnect("renamed", self, "_on_reference_to_renamed")
+			_reference_to.disconnect("minimum_size_changed", self, "minimum_size_changed")
 		_reference_to = control
+		_reference_to.connect("minimum_size_changed", self, "minimum_size_changed")
```

Comment:
This call to `connect` should be done below the check `if not _reference_to:`, alongside where the connection to `renamed` is made, or else we may call a method on a `nil` value.